lib/widgets/message_list.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:math';
+import 'dart:ui';
```

Comment:
These added imports are failing CI because they're unnecessary. (I am a bit curious what caused them to appear in your editing, though.)

Reply:
I was using Flutter version 3.16.9, and I imported certain files due to an error in the 'FontFeature.enable' part of the code, which was causing an unrecognized Target URI error. To resolve this, I had to import them.

However, after upgrading both Flutter and the SDK version, those errors are no longer present.

Reply:
Ah, that makes sense. Yeah, so a good reason to match our Flutter version, then.